packages/template-retail-react-app/app/components/display-price/index.jsx

```diff
@@ -7,61 +7,181 @@
-    return (
-        <Skeleton isLoaded={basePrice} display={'flex'} {...skeletonProps}>
-            <Text fontWeight="bold" fontSize="md" mr={1}>
-                {isProductASet &&
-                    `${intl.formatMessage({
-                        id: 'product_view.label.starting_at_price',
-                        defaultMessage: 'Starting at'
-                    })} `}
-            </Text>
-            {typeof discountPrice === 'number' && (
-                <Text as="b" {...discountPriceProps}>
-                    {intl.formatNumber(discountPrice, {
-                        style: 'currency',
-                        currency: currency || activeCurrency
-                    })}
-                </Text>
-            )}
-            <Text
-                as={typeof discountPrice === 'number' ? 's' : 'b'}
-                ml={typeof discountPrice === 'number' ? 2 : 0}
-                fontWeight={discountPrice ? 'normal' : 'bold'}
-                {...basePriceProps}
-            >
-                {intl.formatNumber(basePrice, {
-                    style: 'currency',
-                    currency: currency || activeCurrency
-                })}
-            </Text>
-        </Skeleton>
+    const listPriceText = intl.formatNumber(price, {
+        style: 'currency',
+        currency
+    })
+
+    return isRange ? (
+        <Text
+            as={as}
+            {...extraProps}
+            aria-label={intl.formatMessage(msg.ariaLabelListPriceWithRange, {
+                listPrice: listPriceText || ''
+            })}
+            color="gray.600"
+        >
+            {listPriceText}
+        </Text>
+    ) : (
+        <Text
+            as={as}
+            {...extraProps}
+            aria-label={intl.formatMessage(msg.ariaLabelListPrice, {
+                listPrice: listPriceText || ''
+            })}
+            color="gray.600"
+        >
+            {listPriceText}
+        </Text>
     )
 }
 
+ListPrice.propTypes = {
+    price: PropTypes.number.isRequired,
+    currency: PropTypes.string.isRequired,
+    as: PropTypes.string,
+    isRange: PropTypes.bool,
+    extraProps: PropTypes.object
+}
+
 DisplayPrice.propTypes = {
-    basePrice: PropTypes.oneOfType([PropTypes.number, PropTypes.string]),
-    discountPrice: PropTypes.number,
-    currency: PropTypes.string,
-    isProductASet: PropTypes.bool,
-    discountPriceProps: PropTypes.object,
-    basePriceProps: PropTypes.object,
-    skeletonProps: PropTypes.object
+    priceData: PropTypes.shape({
+        currentPrice: PropTypes.number.isRequired,
+        isOnSale: PropTypes.bool.isRequired,
+        listPrice: PropTypes.number,
+        isASet: PropTypes.bool,
+        isMaster: PropTypes.bool,
+        isRange: PropTypes.bool,
+        hasRepresentedProduct: PropTypes.bool,
+        maxPrice: PropTypes.number,
+        tieredPrice: PropTypes.number
+    }),
+    currency: PropTypes.string.isRequired
 }
 
 export default DisplayPrice
```

Comment:
If customizability is one of the criteria, how would developers customize the default behaviour of DisplayPrice? 

I think it may be helpful to export more things. So if I want to override CurrentPrice only, I don't need to override the entire file.

```suggestion
export default DisplayPrice
export {ListPrice, CurrentPrice, msg}
```